Tests/AK/TestTrie.cpp

```diff
@@ -6,13 +6,14 @@
 
 #include <LibTest/TestCase.h>
 
+#include <AK/StringView.h>
 #include <AK/Trie.h>
 
 TEST_CASE(normal_behaviour)
 {
     Trie<char, String> dictionary('/', "");
-    constexpr static const char* data[] { "test", "example", "foo", "foobar" };
-    constexpr static const size_t total_chars = 18; // root (1), 'test' (4), 'example' (7), 'foo' (3), 'foobar' (3, "foo" already stored).
+    constexpr StringView data[] { "test", "example", "foo", "foobar" };
+    constexpr size_t total_chars = 18; // root (1), 'test' (4), 'example' (7), 'foo' (3), 'foobar' (3, "foo" already stored).
     for (auto& entry : data) {
         StringView view { entry };
```

Comment:
Should not be needed anymore as `entry` is a StringView already

Reply:
Nice catch.